Task 1

zhulanov-an opened the pull request:
ДЗ № 1

zhulanov-an committed:
ignore .idea folder

zhulanov-an committed:
add project requirenments

zhulanov-an committed:
update project requirenments

zhulanov-an committed:
add test_list_breeds

zhulanov-an committed:
добавил хук

zhulanov-an committed:
добавил тесты АПИ получения данных пород собак

zhulanov-an committed:
добавил документацию

zhulanov-an committed:
мелкие правки

zhulanov-an committed:
добавил jsonschema в requirenments.txt

zhulanov-an committed:
добавил jsonschema в requirenments.txt

zhulanov-an committed:
добавил тесты

zhulanov-an committed:
добавил тесты

zhulanov-an committed:
добавил тесты

zhulanov-an committed:
добавил документацию

zhulanov-an committed:
поправил документацию

zhulanov-an committed:
поправил тест

zhulanov-an committed:
добавил фикстуру

konflic commented on tests/test_dog_ceo.py:
```diff
@@ -0,0 +1,103 @@
+    assert resp_1.ok
+    assert resp_2.ok
+    assert res_1["status"] == res_2["status"] == "success"
+
```
Зачем тут пустая строка?

zhulanov-an replied:
лишний перенос убрал https://github.com/zhulanov-an/pythonQAApi/pull/1/commits/0fdbb3aeb34afe22e06521ec848384f6762180bc

konflic commented on conftest.py:
```diff
@@ -0,0 +1,10 @@
+@pytest.fixture()
+def base_url():
+    return "https://dog.ceo/api"
+
```
Много пустых строк в файле

zhulanov-an replied:
убрал https://github.com/zhulanov-an/pythonQAApi/pull/1/commits/279877d801d7a73881aac39173d490f136031f3b

zhulanov-an committed:
убрал лишний перенос

zhulanov-an committed:
удалил лишние переносы

zhulanov-an committed:
убрал лишний импорт

zhulanov-an committed:
перенос фикстур в тест

zhulanov-an committed:
1. перенос фикстур в тест
2. перенос conf_test в директорию с тестами
2. подготовка проекта к слиянию

zhulanov-an committed:
1. перенос фикстур в тест
2. перенос conf_test в директорию с тестами
2. подготовка проекта к слиянию

zhulanov-an committed:
Merge branch 'task_4' into task_3

# Conflicts:
#	README.md

zhulanov-an committed:
1. перенос фикстур в тест
2. перенос conf_test в директорию с тестами
2. подготовка проекта к слиянию

zhulanov-an committed:
Merge branch 'task_3' into task_2

# Conflicts:
#	README.md

zhulanov-an committed:
1. перенос фикстур в тест
2. перенос conf_test в директорию с тестами
2. подготовка проекта к слиянию

zhulanov-an committed:
Merge branch 'task_2' into task_1

# Conflicts:
#	README.md

zhulanov-an committed:
1. перенос фикстур в тест
2. перенос conf_test в директорию с тестами
3. подготовка проекта к слиянию
4. реализовал параметризацию через хук

zhulanov-an committed:
Merge remote-tracking branch 'origin/task_1' into task_1

# Conflicts:
#	conftest.py
